File: python/character_match.py
def is_character_match(string1, string2):
    isAnagram = True
    lowerString1 = string1.lower()
    lowerString2 = string2.lower()
    for i in lowerString1:
        if i not in lowerString2:
            isAnagram = False
            break
    return isAnagram

File: js/characterMatch.js
exports.isCharacterMatch = function (string1, string2) {
  let isAnagram = true;
  let lowerString1 = string1.toLowerCase();
  let lowerString2 = string2.toLowerCase();
  for (let i of lowerString1) {
    if (!lowerString2.includes(i)) {
      isAnagram = false;
      break;
    }
  }
  return isAnagram;
};
